modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java

```diff
@@ -467,6 +467,13 @@ public Organization postOrganization(Organization organization)
 				_getWebsites(organization, null),
 				_createServiceContext(organization));
 
+		byte[] logoBytes = _getLogoBytes(organization, null, false);
+
+		if (ArrayUtil.isNotEmpty(logoBytes)) {
+			_organizationService.updateLogo(
```

Comment:
I think it's better to save the organization returned by this
`serviceBuilderOrganization = _organiza...`